### ISSUE TITLE
Operate on selected text

### ISSUE THREAD

**Issue by JamieMason** [swirlycheetah/jasmine-scaffold-sublime-text#2]
Hi mate,
Cheers for the great plugin, really saves time. I have a small request based on how some plugins like "JS Format" work;

```
when no text is selected
    format the whole file
when text is selected
    format selected text only
```

It'd be handy if this worked the same way, so that in an existing suite I can scaffold out additional specs then have them exploded in place.

**Comment by chrisdwheatley** [comment]
Good idea! :+1: I'll take a look at implementing this shortly.

**Comment by jseto** [comment]
I do support this enhancement!!!

**Comment by chrisdwheatley** [comment]
This is fixed and was released as part of [0.1.0](https://github.com/swirlycheetah/jasmine-scaffold-sublime-text/releases/tag/0.1.0). If you notice any issues please let me know.

**Comment by JamieMason** [comment]
Fanastastic, thanks a lot for doing this @swirlycheetah. I use this plugin all the time and this will be a real help.
